x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/json_diff/diff_view.tsx

```diff
@@ -140,50 +141,75 @@ const convertToDiffFile = (oldSource: string, newSource: string) => {
 const TABLE_CLASS_NAME = 'rule-update-diff-table';
 const CODE_CLASS_NAME = 'rule-update-diff-code';
 const GUTTER_CLASS_NAME = 'rule-update-diff-gutter';
+const DARK_THEME_CLASS_NAME = 'rule-update-diff-dark-theme';
 
 const CustomStyles: React.FC = ({ children }) => {
   const { euiTheme } = useEuiTheme();
 
   const customCss = css`
-    .${TABLE_CLASS_NAME} .diff-gutter-col {
-      width: ${euiTheme.size.xl};
+    .${CODE_CLASS_NAME}.diff-code {
+      padding: 0 ${euiTheme.size.l} 0 ${euiTheme.size.m};
     }
 
-    .${CODE_CLASS_NAME}.diff-code, .${GUTTER_CLASS_NAME}.diff-gutter {
-      background: transparent;
+    .${TABLE_CLASS_NAME} .diff-gutter-col {
+      width: ${euiTheme.size.xl};
     }
 
+    /* Vertical line separating two sides of the diff view */
     .${GUTTER_CLASS_NAME}:nth-child(3) {
       border-left: 1px solid ${euiTheme.colors.mediumShade};
     }
 
+    /* Gutter of a line with deletions */
     .${GUTTER_CLASS_NAME}.diff-gutter-delete {
-      color: ${euiTheme.colors.dangerText};
       font-weight: bold;
+      background: rgb(255, 215, 213);
+    }
+    .${DARK_THEME_CLASS_NAME} .${GUTTER_CLASS_NAME}.diff-gutter-delete {
+      background: rgba(248, 81, 73, 0.3);
```

Comment:
I'm guessing none of these colors are part of the eui theme presets? It might be more readable to abstract them out to constants honestly, but I can understand with the comments you've added.

Reply:
These are slightly tweaked colors from GitHub's diff view. I think extracting these into constants is a good idea.